src/components/Header/Header.tsx

```diff
@@ -50,30 +49,48 @@ const Header = ({ nav, user }: Props) => {
           </NextLink>
         )}
       </div>
-      {nav !== undefined && (
+      {
         <nav className={` bg-black px-4 py-2 ${showNav ? "nav--open" : ""}`}>
-          <ul
+          <div
             className={`${
               showNav ? "flex-col" : "flex-row "
             }  flex justify-around`}
           >
-            {nav.map((item) => {
```

Comment:
Why did you remove this `nav` prop and decided to put the values here?

You already had the varialbe inside `Constants.ts` that was being passes. You should have modified that... Please revert this change.